src/Plugin/MediaEntity/VideoProvider/YouTube.php

```diff
@@ -44,24 +86,14 @@ public function defaultConfiguration() {
    * {@inheritdoc}
    */
   public function thumbnailURI() {
-    $maxres_thumb = 'http://img.youtube.com/vi/' . $this->matches['id'] . '/maxresdefault.jpg';
```

Comment:
FYI this is how I get the thumbnail URL in a library that I maintain https://github.com/jcisio/MediaWrapper/blob/master/MediaWrapper/Wrapper/Youtube.php#L55 (maxresdefault, then mqdefault). Again, I don't have source documenting it...